packages/console-sdk-microfrontend/src/index.ts

```diff
@@ -24,9 +24,9 @@ export type {
   IConsoleProps,
   IMicrofrontendIntegrator,
   Events,
-  EventsTypes,
 }
 
 export {
   ConsoleSDK,
+  EventsTypes,
```

Comment:
why has this been moved?

Reply:
it makes sense since it is not a type but an enum, just want to make sure this is intended :)